ivy_tests/test_ivy/test_frontends/test_jax/test_devicearray.py

```diff
@@ -324,6 +324,38 @@ def test_jax_devicearray_conj(
         on_device=on_device,
     )
 
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="jax.numpy.array",
+    method_name="imag",
+    dtype_and_x=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("real_and_complex"),
+    ),
+)
+def test_jax_devicearray_imag(
+    dtype_and_x,
+    on_device,
+    frontend,
+    frontend_method_data,
+    init_flags,
+    method_flags,
+):
+    input_dtype, x, axis = dtype_and_x
+    helpers.test_frontend_method(
+        init_input_dtypes=input_dtype,
+        init_all_as_kwargs_np={
+            "object": x[0],
+        },
+        method_input_dtypes=input_dtype,
+        method_all_as_kwargs_np={},
+        frontend=frontend,
+        frontend_method_data=frontend_method_data,
+        init_flags=init_flags,
+        method_flags=method_flags,
+        on_device=on_device,
```

Comment:
`backend_to_test = backend_fw`

Can you add this line to the to the `test_frontend_method`? This has been added in a refactor, the test_frontend_function and the test_frontend_function both has this argument now.